styles/styles.css

```diff
@@ -683,6 +683,18 @@ main .section.no-margin-bottom {
   margin-bottom: 0;
 }
 
+/* section - add pidding 1rem */
+main .section.padding-1 {
+  padding: 0 1rem;
+}
+
+/* Avoid double section-container styling for fragment */
+main .section.fragment-container > .section-container {
```

Comment:
Do fragments come wrapped in their own sections? @jindaliiita , I remember we discussing this. In this case, should we unwrap the fragments so that such double styling is not applied?

Reply:
Yes i think unwrapping of fragments should be good as i think this is creating problems at various places

Reply:
I am not sure if i have get your point correctly but i am thinking of modifying fragment block and removing one section container. WDYT ?

Reply:
I have checked the block collection code.
section-container creation code is not present there. is this hierarchy removed from sections ? @sdmcraft any idea
https://github.com/adobe/helix-block-collection/blob/main/scripts/lib-franklin.js#L247

Reply:
If we can remove one `.section-container`, that would be the best solution.

Reply:
I think the problem is this nesting of `section-container` in `fragment`

![image](https://github.com/hlxsites/sunstar/assets/1191451/a8538325-e967-4186-9ee9-8e4516b8e508)

@jindaliiita , can this be avoided?

Reply:
i am looking into this